files/image_config/syslog/host_umount.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#This script is invoked at the closure of syslog socket during reboot
+#This will stop journal services, unmount /var/log and delete loop device 
+#associated to /host to ensure proper unmount of /host
+
+journal_stop() {
+    systemctl stop systemd-journald.service
+    systemctl stop systemd-journald.socket
+    systemctl stop systemd-journald-audit.socket
+    systemctl stop systemd-journald-dev-log.socket
+}
+
+delete_loop_device() {
+    umount /var/log
+    if [[ $? -ne 0 ]]
+    then
+	exit 0
+    fi
+    losetup -d /dev/loop1
+}
+
+case "$1" in
+    journal_stop|delete_loop_device)
+	$1
+	;;
+    *)
+	echo "Usage: $0 {journal_stop|delete_loop_device}"
```

Comment:
>echo [](start = 1, length = 4)

echo to stderr, and exit with a non-zero code

Reply:
Hi Qi,

This is for line number 17 or 27?

Reply:
For this line
```
echo "Usage: $0 {journal_stop|delete_loop_device}"
```

---
In reply to: [423504781](https://github.com/Azure/sonic-buildimage/pull/4558#discussion_r423504781) [](ancestors = 423504781)

Reply:
Accepted